src/assets/i18n/de_DE.json

```diff
@@ -571,6 +566,7 @@
   "account.wishlists.no_items": "Sie haben keine verfügbaren Artikel zu Ihrer Wunschliste hinzugefügt.",
   "account.wishlists.no_wishlists": "Derzeit haben Sie keine Wunschlisten.",
   "account.wishlists.table.preferred": "Standardliste",
```

Comment:
See line 556: Wenn Sie eine Wunschliste zu Ihrer bevorzugten Wunschliste machen, werden alle anderen Listen (sofern vorhanden) auf Standard zurückgesetzt.

So, the translation "Standardliste" seems to be wrong here. Should be "Bevorzugt". French translation is fine. I'll adjust it for DE as well now.